components/upload/UploadList.tsx

```diff
@@ -170,13 +194,17 @@ export default class UploadList extends React.Component<UploadListProps, any> {
       const linkProps =
         typeof file.linkProps === 'string' ? JSON.parse(file.linkProps) : file.linkProps;
 
-      const removeIcon = showRemoveIcon ? (
+      const removeIcon = showRemoveIcon ? customRemoveIcon && (
+        this.handleActionIconRender(customRemoveIcon, () => this.handleClose(file), locale.removeFile)
```

Comment:
Duplicate title

Reply:
The origin code has title props in `<DownloadOutlined />` and `<DeleteOutlined />` as well. Should I remove them? I find no such API in antd Icon component.

Reply:
Judging from the origin snapshot on master branch, `title` works in Icon component and the duplicate title problem seems to be an old problem. Not only custom Icon has duplicate title, but also origin Icon. Should I fix them both?

Reply:
No, you are right.